### PR TITLE
LPS-184467 Implement a fragment showing FDS View v4

### PR DESCRIPTION
**Pull request by dsanz**
I've rebased @markocikos work (https://github.com/liferay-frontend/liferay-portal/pull/3389) on top of current master

Let's see what CI says...

### CODE REVIEW

**Comment by dsanz** [comment]
ci:test:stable

**Comment by dsanz** [comment]
ci:relevant

**Comment by dsanz** [comment]
ci:test:relevant

**Comment by dsanz** [comment]
ci:test:stable

**Comment by dsanz** [comment]
ci:test:relevant

**Comment by dsanz** [comment]
ci:test:relevant

**Comment by dsanz** [comment]
ci:sf

**Comment by dsanz** [comment]
ci:test:sf

**Comment by dsanz** [comment]
ci:test:relevant

**Comment by dsanz** [comment]
ci:test:stable

**Comment by dsanz** [comment]
ci:test:relevant

**Comment by dsanz** [comment]
ci:test:stable

**Comment by dsanz** [comment]
@CarlosBrichete We need a quick fix over [this test](https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3157/functional-tomcat90-mysql57-chrome100-jdk8/23/20/LocalFile.DatasetsDateFilters_CannotSaveAnInvalidDateRangeInDateFields/index.html.gz?authuser=0) and check the other 9 failing tests to forward this. 

We need to send this on monday to keep on Q2 schedule, please prioritize this.
cc @manoelcyreno 

Thank you

**Comment by dsanz** [comment]
ci:test:relevant

**Comment by markocikos** [comment]
Out of four unique failures in https://github.com/liferay-frontend/liferay-portal/pull/3422#issuecomment-1586189774:
- Carlos just quarantined (3)
- I don't understand (1) and (4), they look flaky and unrelated
- I tested (2) and could not reproduce it. Create button is clearly on the page:
screen1

**Comment by markocikos** [comment]
https://github.com/liferay-frontend/liferay-portal/pull/3422#issuecomment-1587157885 failures look unrelated, caused by kernel semver errors :disappointed:

**Comment by dsanz** [comment]
Thanks @CarlosBrichete @markocikos, so, can we say this PR has unrelated failures at this moment?

**Comment by markocikos** [comment]
I'm not sure. It looks like kernel got updated in `master`, I'll try again.

**Comment by markocikos** [comment]
ci:forward

**Comment by CarlosBrichete** [comment]
ci:forward

**Comment by CarlosBrichete** [comment]
Sorry, I put the ignore property wrong :(

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DataSetsDateFilters.testcase [hunk 1]
```diff
@@ -195,6 +195,7 @@ definition {
 
 	@description = "LPS-181281. Confirms the user cannot create a new date filter with invalid date range."
 	@priority = 4
+	@ignore = "true"
```
Comment (markocikos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DataSetsDateFilters.testcase:198]: Needs to be before priority? :thinking:

Comment (CarlosBrichete) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DataSetsDateFilters.testcase:198]: oh yes, thats true

**Comment by CarlosBrichete** [comment]
ci:forward

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
Hi @dsanz I realized that the failure that SF returns is not related. 

When I ran source format locally it updated a .jsp file, but I don't know if I can make this change since it is not related to the poshi tests I sent a new PR with this change and to advance some CI results here: https://github.com/liferay-frontend/liferay-portal/pull/3435

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by dsanz** [comment]
temp pull for renaming tests: https://github.com/liferay-frontend/liferay-portal/pull/3435

**Comment by markocikos** [comment]
Followup in https://github.com/liferay-frontend/liferay-portal/pull/3439

**Pull request closed without merging**